save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/test/TestFilesContent.kt

```diff
@@ -0,0 +1,19 @@
+/**
+ * DTOs for public test transfer
+ */
+
+package com.saveourtool.save.test
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property testLines public test's test file lines
+ * @property expectedLines public test's expected file lines
+ * @property tags list of tags of current test
+ */
+@Serializable
+data class TestFilesContent(
```

Comment:
we are able to transfer files as `FilePart` in binary format. see `DownloadFilesController` for example.
it will allow to transfer files in common way and read it only where it's required

Reply:
> FilePart is jvm

No, `FilePart` is a way to describe common HTTP concept of `multipart` content type

Reply:
> For those who use our API it might be hard enough to read `TestFilesContent` if files are `FilePart`.

Agree. Also, I don't see where we use `/public-test` 

> Moreover as far as I understand, `FilePart` is jvm, isn't it?

I'm not sure, but orchestrator uses it to upload user's files to backend